src/vcf/parsing_state.cpp

```diff
@@ -87,5 +91,31 @@ namespace ebi
     {
         defined_metadata.emplace(meta_type, id);
     }
+
+    void ParsingState::validate_additional_checks()
+    {
+        if (additionalChecks.checkEvidence) {
+            if (genotypes_present() || allele_frequencies_present()) {
```

Comment:
I forgot: it's ok too if the VCF only has allele counts (AC and AN). The frequency can be calculated from that.